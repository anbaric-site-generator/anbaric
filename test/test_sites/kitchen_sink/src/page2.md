---
template: layout-two.html.hbs
---
# This is page 2
Are we testing? {{ site.testing }}