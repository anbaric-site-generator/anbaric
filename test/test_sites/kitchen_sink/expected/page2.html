<html>
  <head></head>
  <body>
    <h2>This is layout 2</h2>
    <h1>This is page 2</h1>
  </body>
</html>